src/dir.props

```diff
@@ -1,3 +1,4 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="BuildNumber.props" />
   <Import Project="..\dir.props" />
```

Comment:
Is there any reason this is only under src/dir.props? Similar question for the BuildNumber.props why not put it in the root of the repo?